snake-py/snake.py

```diff
@@ -0,0 +1,56 @@
+from turtle import *
+from random import randrange
+from freegames import square, vector
+
+food = vector(0, 0)
+snake = [vector(10, 0)]
+aim = vector(0, -10)
+
+def change(x, y):
+    "Change snake direction."
+    aim.x = x
+    aim.y = y
+
+def inside(head):
+    "Return True if head inside boundaries."
+    return -200 < head.x < 190 and -200 < head.y < 190
+
+def move():
+    "Move snake forward one segment."
+    head = snake[-1].copy()
+    head.move(aim)
+
+    if not inside(head) or head in snake:
+        square(head.x, head.y, 9, 'red')
+        update()
+        return
+
+    snake.append(head)
+
+    if head == food:
+        print('Snake:', len(snake))
+        food.x = randrange(-15, 15) * 10
+        food.y = randrange(-15, 15) * 10
+    else:
+        snake.pop(0)

```

Comment:
document the code, then code climate issues might get fixed.
cognitive complexity means difficult to read code.

So using appropriate variable names(your code has though) and proper line spaces will do the job